R/rintrinio.R

```diff
@@ -36,8 +36,85 @@ gather_financial_statement_time_series <- function(api_key, ticker, statement, y
 #' @examples
 #' gather_financial_statement_company_compare(api_key, ['AAPL', 'CSCO'], 'income_statement', '2019', 'Q1')
 
-gather_financial_statement_company_compare <- function(api_key, ticker, statement, year, period) {
-  tibble()
+gather_financial_statement_company_compare <- function(api_key, ticker, statement, year, period){
+  
+  #check if the api_key is a string
+  if (typeof(api_key) != "character"){
+    stop("the api_key must be a string")
+  }
+  
+  #check if the ticker is a string
+  if (typeof(ticker) != "character") {
+    stop("the ticker must be a string")
+  }
+  
+  #check if the statement is a string
+  if (typeof(statement) != "character") {
+    stop("the statement must be a string")
+  }
+  
+  #check if the year is a string
+  if (typeof(year) != "character") {
+    stop("the year must be a string")
+  }
+  
+  #check if the period is a string
+  if (typeof(period) != "character") {
+    stop("the period must be a string")
+  }
+  
```

Comment:
The logic of the tests above is all good, but I believe that the ticker should be a character vector - can you confirm that this is the logic?